test/functional/sqlsrv/TC46_FetchNextResult.phpt

```diff
@@ -66,10 +65,10 @@ function fetchFields()
             }
         }
     }
-    if (sqlsrv_next_result($stmt1) ||
-        sqlsrv_next_result($stmt2)) {
+    if ($r1 = sqlsrv_next_result($stmt1) ||
+        $r2 = sqlsrv_next_result($stmt2)) {
```

Comment:
why do you need to introduce $r1 and $r2?

Reply:
It was for debugging when I thought I discovered a bug... forgot to remove them afterwards ;)